test/e2e/data/capi-operator/capi-providers-oci.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: capd-system
+---
+apiVersion: turtles-capi.cattle.io/v1alpha1
+kind: CAPIProvider
+metadata:
+  name: docker
+  namespace: capd-system
+spec:
+  type: infrastructure
+  name: docker
+  version: "{{ .ProviderVersion }}"
+  fetchConfig:
+    oci: registry.rancher.com/rancher/cluster-api-operator-components:"{{ .ProviderVersion }}"
```

Comment:
```suggestion
    oci: registry.rancher.com/rancher/cluster-api-operator-components:{{ .ProviderVersion }}
```
or maybe trim the version string in code

Reply:
thanks for checking this, I have applied changes